package.json

```diff
@@ -3,6 +3,7 @@
     "template": "node addRequestTemplate.js"
   },
   "devDependencies": {
+    "json-schema-ref-parser": "^9.0.9",
```

Comment:
We should use the better maintained version of this library -- https://www.npmjs.com/package/@apidevtools/json-schema-ref-parser

This was the suggestion of the library author on the npm page
<img width="985" alt="image" src="https://github.com/user-attachments/assets/ae8451b1-b636-44e0-9cd8-6f044b563feb">

Reply:
Sounds good I’ll add that as a follow up